### ISSUE TITLE
`get_storms` not pulling 2017

### ISSUE THREAD

**Issue by timtrice** [ropensci/rrricanes#18]
As of now Tropical Storm Arlene exists in the archives:

http://www.nhc.noaa.gov/archive/2017/

However, the call:

```r
storms <- get_storms(year = lubridate::year(Sys.Date()), basin = c("AL", "EP"))
```

Returns the following error:

```
Error in FUN(X[[i]], ...) : There are no storms for year 2017
```

**Comment by timtrice** [comment]
There is an empty <var>td</var> element currently on the archive page for Eastern Pacific which has not yet had storms. In `.extract_storms()` it picks up the empty element and expects another storm that doesn't exist. 

On the check of `length(col.names) == 0`, the empty element forces the error. The following call, 

```r
get_storms(basin = "AL")
```

executes as expected and returns Arlene. But since there are not yet any EP storms and the default call looks for both AL and EP storms, we get the error.

**Comment by timtrice** [comment]
Error downgraded to a message; allows continuation of collection. 

Resolved in commit 4dd3c31.
